src/Skeletor/Frameworks/Framework.php

```diff
@@ -86,11 +87,11 @@ public function setVersion(string $version)
      */
     public function getPath(string $path)
     {
-        if(array_key_exists($path, $this->paths)){
-            return $this->paths[$path];
+        if(array_key_exists($path, $this->paths) === false){
```

Comment:
@kielabokkie I find it can be tricky, cos if you have the value on the right hand side, that means a silly mistake like $var = false (just one equal) is valid syntax and hard to debug. Either way, we should be consistent, in other place we have if(!class_exists($packageClass)) {